docs/tutorial/named-entity-standardization.md

```diff
@@ -0,0 +1,22 @@
+# Named Entity Standardization
```

Comment:
```suggestion
# Named Entity Standardization

<a href="https://oboacademy.github.io/obook/reference/obook-maturity-indicator/"><img src="https://img.shields.io/endpoint?url=https%3A%2F%2Fraw.githubusercontent.com%2FOBOAcademy%2Fobook%2Fmaster%2Fdocs%2Fresources%2Fobook-badge-development.json" /></a>
```

Reply:
Just to avoid long discussion about alternative approaches here

Reply:
I use these maturity indicator badges on all my tutorials, like https://oboacademy.github.io/obook/howto/are-two-entities-the-same/